src/scala/tiefvision-web/conf/application.conf

```diff
@@ -11,6 +11,6 @@ play.i18n.langs = [ "en" ]
 # Database configuration
 slick.dbs.bounding_box.driver="slick.driver.H2Driver$"
 slick.dbs.bounding_box.db.driver="org.h2.Driver"
-slick.dbs.bounding_box.db.url="jdbc:h2:tcp://localhost/~/tiefvision/db/tiefvision"
-slick.dbs.bounding_box.db.user="sa"
-slick.dbs.bounding_box.db.password=""
\ No newline at end of file
+slick.dbs.bounding_box.db.url="jdbc:h2:tcp://localhost/~/tiefvision"
+slick.dbs.bounding_box.db.user=""
```

Comment:
Why not just remove `user` and `password` entirely from the config? If anything, you're likely to get a configuration error at runtime - more clearly telling someone to define those - instead of "invalid username and password".